src/ui/public/filter_bar/filter_bar.html

```diff
@@ -77,6 +81,21 @@
       </div>
     </div>
 
+    <div
+      class="filter-nav-link__icon"
+      tooltip="{{ filterNavToggle.tooltipContent }}"
+      tooltip-placement="left"
+      tooltip-popup-delay="0"
+      tooltip-append-to-body="1"
+      ng-show="filters.length && showCollapseLink()"
+      ng-class="filterNavToggle.isOpen == true ? '' : 'filter-nav-link--close'"
+      aria-hidden="!filters.length || !showCollapseLink()"
+    >
+      <a class="filter-nav-link__anchor" ng-click="toggleFilterShown($event)">
```

Comment:
Can we just use the existing circle-down font-awesome icon, similar to how we use the circle-left icon for the sidebar? https://fontawesome.com/icons/chevron-circle-down?style=solid 

cf. https://github.com/elastic/kibana/blob/5cddc100778f0dfd5c2e1f1342ab40a1cc7f52b9/src/ui/public/collapsible_sidebar/collapsible_sidebar.js#L30-L30

Reply:
Originally I thought I should use a similar icon on the app-switcher sidebar. It makes sense to avoid duplicates. I will switch to font-awesome.